alembic/versions/xxxx_initial_schema.py

```diff
@@ -0,0 +1,103 @@
+"""initial schema
+
+Revision ID: xxxx
+Revises: 
+Create Date: 2024-xx-xx
+"""
+from alembic import op
+import sqlalchemy as sa
+
+revision = 'xxxx'
```

Comment:
Could please fill the missing information here, both `Create Date: ..` and `revision = '..'` must be defined, you can take the previous migration file as example. The revision can be a random ID in this case.